### ISSUE TITLE
Add different charting options for presentation of results

### ISSUE THREAD

**Issue by dazzasmith** [matterpoll/matterpoll#254]
#### Summary

Add one or more different charting options for presentation of poll results (e.g. histogram, pie chart etc.). This would greatly benefit decision makers, and allow better and more speedy visualization of results.

#### Describe alternatives you've considered

Perhaps an alternative would be to integrate the results with as input to another existing charting plugin.

#### Additional context

Currently, there is just a single way, not so digestible way of presenting the results of the poll to the user:
![poll](https://user-images.githubusercontent.com/7801013/72978120-1bfbbd80-3dd6-11ea-94a9-59e5bbc13fda.PNG)

**Comment by kaakaa** [comment]
Thank you for commenting @dazzasmith.
@tawalaya shows us the simple chart option https://github.com/matterpoll/matterpoll/pull/251. How does this sound?

**Comment by dazzasmith** [comment]
> Thank you for commenting @dazzasmith.
> @tawalaya shows us the simple chart option #251. How does this sound?

Thanks for your reply kaakaa. It'd certainly be a very good start, making results more accessible.

**Comment by hanzei** [comment]
@dazzasmith Would you be fine if this issues gets closed a duplicate of https://github.com/matterpoll/matterpoll/issues/177?

**Comment by dazzasmith** [comment]
Yes, I think so. Are there any plans to add one or two other options? For example a pie chart would be a very welcome alternative form of visualization.

**Comment by hanzei** [comment]
How would a user (or the admin) decide what kind of chart is wanted?

**Comment by dazzasmith** [comment]
--progress-pie perhaps?

**Comment by hanzei** [comment]
I've found https://github.com/wcharczuk/go-chart which might be a good library to generate the svg charts.
